client/odie/utils/async-load.jsx

```diff
@@ -0,0 +1,71 @@
+import PropTypes from 'prop-types';
+import { Component } from 'react';
+
+import './style.scss';
+
+function omit( obj, fields ) {
+	const newObj = { ...obj };
+	for ( const field of fields ) {
+		if ( newObj.hasOwnProperty( field ) ) {
+			delete newObj[ field ];
+		}
+	}
+	return newObj;
+}
```

Comment:
This is the result of removing `lodash` library. It was included as a dependency in order to load it's `omit` function